docs/book/src/topics/consuming-existing-azure-infrastructure.md

```diff
@@ -0,0 +1,6 @@
+# Consuming Existing Azure Infrastructure
```

Comment:
It might be good to stick to "Externally Managed Infrastructure" so it doesn't get confused with pre-existing resources like BYO vnet and resource group